src/sql/platform/connection/test/common/testNotificationService.ts

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { INotification, INotificationHandle, INotificationService, IPromptChoice, IPromptChoiceWithMenu, IPromptOptions, IStatusMessageOptions, NotificationMessage, NotificationsFilter, Severity } from 'vs/platform/notification/common/notification';
+import { Event, IDisposable } from 'vs/workbench/workbench.web.main';
+
+export class TestNotificationService implements INotificationService {
```

Comment:
This already exists - src\vs\platform\notification\test\common\testNotificationService.ts